pages/common/chainctl.md

```diff
@@ -0,0 +1,36 @@
+# chainctl
+
+> The official CLI for Chainguard.
+> More information: <https://edu.chainguard.dev/chainguard/chainctl>.
+
+- Authenticate to the Chainguard Platform:
+
+`chainctl auth login`
+
+- Logout from the Chainguard Platform:
+
+`chainctl auth logout`
+
+- Check which version you have installed:
+
+`chainctl version`
+
+- Update to the latest version:
+
+`chainctl update`
+
+- List images available to your account:
+
+`chainctl images list`
+
+- List image repositories available to your account:
+
+`chainctl images repos list`
+
+- Examine the history of an image tag in chainctl (e.g., IMAGE=python TAG=3):
+
+`chainctl images history {{image}}:{{tag}}`
+
+- List package version data from repositories available to your account (e.g., PACKAGE_NAME=go):
+
+`chainctl packages versions list {{package_name}}`
```

Comment:
```suggestion
- Examine the history of an image tag in chainctl:

`chainctl images history {{python}}:{{3}}`

- List package version data from repositories available to your account:

`chainctl packages versions list {{go}}`
```
This might be better. What do you think?